CDTDatastoreTests/CDTReplicationTests.m

```diff
@@ -26,10 +26,17 @@
 #import "TD_Revision.h"
 #import "TDPuller.h"
 #import "TDPusher.h"
+#import "CDTSessionCookieInterceptor.h"
 #import <OHHTTPStubs/OHHTTPStubs.h>
 #import <OHHTTPStubs/OHHTTPStubsResponse+JSON.h>
 #import <OCMock/OCMock.h>
 
+
+@interface CDTAbstractReplication (Interceptors)
+//expose the interceptor property for inspection.
+@property (nonnull, nonatomic, readwrite, strong) NSArray *httpInterceptors;
```

Comment:
Fixed in c2d0cfb. It turns out that the interceptors are available on the public interface as readonly anyway